### PR TITLE
Update config.toml for dev-1.22 branch.

### PR DESCRIPTION
**Pull request by PI-Victor**
This will bump the versions of the `config.toml` according to the latest release patches for each individual version.
It also replaces the `master` branch with `main` 

/assign @jimangel @irvifa @sftim 

~p.s.: There is no `release-1.21` branch yet, this will be created on release day manually.~
release-1.21 was created!

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/approve

File: config.toml [hunk 1]
```diff
@@ -181,30 +181,37 @@ js = [
...
+githubbranch = "v1.19.14"
 docsbranch = "release-1.19"
 url = "https://v1-19.docs.kubernetes.io"
 
+[[params.versions]]
+fullversion = "v1.18.20"
+version = "v1.18"
+githubbranch = "v1.18.20"
+docsbranch = "release-1.18"
+url = "https://v1-18.docs.kubernetes.io"
```
Comment (PI-Victor) [inline config.toml:213]: i've added the 1.18.20 branch back to the list

**Comment by PI-Victor** [comment]
we can go ahead and merge this now, since it's targeting the dev branch.

/hold cancel

**Pull request merged**